doc-site/docs/reference/types/_includes/transactionreceiptlistener_description.md

```diff
@@ -20,7 +20,11 @@
 }
 ```
 
-Note the ability to filter on particular receipt types, and most importantly the ability to block delivery on a given contract address while a "gap" exists in the available states.
+Note the ability to filter on particular receipt types, and most importantly the ability to control delivery behavior when states are incomplete:
+
+- **`block_contract`** (default): Pauses delivery for each individual smart contract address when incomplete states are detected
```

Comment:
Really unsure what our default should be... but I guess in the absence of a clearly "better" choice, we just stick with what it's been. All of them are problematic if you don't understand what you're asking for.